### PR TITLE
Feature/boot 44 set up theme colours

### PR DESCRIPTION
**Pull request by matty-charlton**
[BOOT-44](https://bootbag.atlassian.net/browse/BOOT-44)

- Add the style guide colours to the global variable scss file
- Fix styleResources config with correct path and scss key allowing the use of global scss variables within vue styles
- Add `$negative-red` to error colour attribute

### CODE REVIEW

File: assets/scss/vars.scss
Comment (DaisyyWard) [inline assets/scss/vars.scss:1]: Hi Matty, I was thinking more along the lines of something like this

`@import "~bulma/sass/utilities/_all";

// Brand Colours
$brand-pink: #E30285;
$brand-pink-invert: findColorInvert($brand-pink);
$brand-turquoise: #008080

// Background Colours
$background-dark: #020202;
$background-medium: #2C2C3E;

// Leagues
$league-qualifier: #4455DA;
$league-developer: #C75300;
$league-challenger: #008558;
$league-elite: $brand-turquoise;
$league-premier: #8A7500;

// Signage
$positive-green: #01E437;
$alert-yellow: #FFEF02;
$negative-red: #FF5C5F;
$notification-red: #EA0000;
$info: $brand-pink;

// Links
$link: $white;
$link-invert: $grey;
$link-focus-border: $brand-turquoise;

// Other
$body-text-color: $white;
$background-primary: $dark-bg;

// Basic
$white: #FFFFFF;`

It's not perfect but I feel like it's more scalable with the prefixes. 

I haven't looked at the designs so where I changed turquoise to a brand colour I am not sure about if that's right or not

Where is the $grey variable coming from?

Comment (matty-charlton) [reply assets/scss/vars.scss:1]: @DaisyyWard I've pushed the changes to the variable names, I'll add the buttons section once we implement the style guide

File: assets/scss/vars.scss [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
-$info: $primary;
+$info: $brand-pink;
 $link-invert: $grey;
-$link-focus-border: $turquoise;
+$link-focus-border: $brand-turquoise;
 
 $body-color: $white;
 
-$background: $black;
+$background: $background-dark;
```
Comment (DaisyyWard) [inline assets/scss/vars.scss:32]: Do we need the $background variable? We could just use $background-dark directly?

Or possibly we could do 

`$background-primary: $background-dark
$background-secondary:  $background-medium`

Comment (matty-charlton) [reply assets/scss/vars.scss:32]: @DaisyyWard Yep we need this, `$background` is a general Bulma variable which we're overriding to be `$background-dark`

Comment (DaisyyWard) [reply assets/scss/vars.scss:32]: Ah okay I didn't know that

Comment (matty-charlton) [reply assets/scss/vars.scss:32]: @DaisyyWard I've added the comment above the Bulma variables

**Review by DaisyyWard — APPROVED** [review]

**Pull request merged**